azurelinuxagent/common/protocol/wire.py

```diff
@@ -369,10 +369,14 @@ def get_blob_type(self, url):
         logger.verbose("Check blob type.")
         timestamp = time.strftime("%Y-%m-%dT%H:%M:%SZ", time.gmtime())
         try:
-            resp = self.client.call_storage_service(restutil.http_head, url, {
-                "x-ms-date": timestamp,
-                'x-ms-version': self.__class__.__storage_version__
-            })
+            resp = self.client.call_storage_service(
+                restutil.http_head,
+                url,
+                {
+                    "x-ms-date": timestamp,
+                    'x-ms-version': self.__class__.__storage_version__
+                },
+                chk_proxy=True)
```

Comment:
@hglkrijger Could we add this to the call_storage_service method directly? That way, it will always be present vs. having to ensure every caller passes.